static/app/utils/profiling/profile/continuousProfile.tsx

```diff
@@ -195,3 +198,27 @@ function getThreadData(profile: Profiling.ContinuousProfile): {
     threadName: thread_metadata?.[threadId]?.name ?? `Thread ${threadId}`,
   };
 }
+
+export function minTimestampInChunk(
+  chunk: Readonly<Profiling.ContinuousProfile>,
+  measurements?: Readonly<Profiling.ContinuousMeasurements>
+): number {
+  let timestamp: number | null = null;
+
+  for (let i = 0; i < chunk.samples.length; i++) {
+    const sample = chunk.samples[i]!;
+    if (!defined(timestamp) || timestamp > sample.timestamp) {
+      timestamp = sample.timestamp;
+    }
+  }
```

Comment:
Can these be out of order?

Reply:
I think we're just assuming it's in order and no guarantees. But it *should* be? We can follow up on this to make sure it's sorted.